src/control/content/js/listView/introMap.js

```diff
@@ -1,5 +1,6 @@
 /* eslint-disable max-len */
 import state from "../../state";
+import isCameraControlVersion from "../../../../shared/utils/cameraControlVersion";
```

Comment:
1- map-utils.js
2- add description to that file
3- import { isCameraControlVersion } from ""...